frontend/public/components/storage-class.tsx

```diff
@@ -12,7 +12,15 @@ const { common } = Kebab.factory;
 const menuActions = [...common];
 
 const defaultClassAnnotation = 'storageclass.kubernetes.io/is-default-class';
-const isDefaultClass = (storageClass: K8sResourceKind) => _.get(storageClass, ['metadata', 'annotations', defaultClassAnnotation], 'false');
+const betaDefaultStorageClassAnnotation = 'storageclass.beta.kubernetes.io/is-default-class';
+const isDefaultClass = (storageClass: K8sResourceKind) => {
+  if ( _.get(storageClass, ['metadata', 'annotations', defaultClassAnnotation], 'false') === 'true' ) {
+    return 'true';
+  } else if ( _.get(storageClass, ['metadata', 'annotations', betaDefaultStorageClassAnnotation], 'false') === 'true' ) {
+    return 'true';
+  }
+  return 'false';
+};
```

Comment:
Make this return a boolean, then you can have the caller format it for displayed as needed. You should be able to reuse this in the storage class dropdown. We probably want to move this to `public/module/k8s/storage-class.ts` if this will be used in more than one place.

```ts
const isDefaultClass = (storageClass: K8sResourceKind) => {
  const annotations = _.get(storageClass, 'metadata.annotations') || {};
  return annotations[defaultClassAnnotation] === 'true' || annotations[betaDefaultStorageClassAnnotation] === 'true';
};

Reply:
Thanks @spadgett , it looks nicer. I've updated the code. Should we move `isDefaultclass` to `public/module/k8s/storage-class.ts`?

Reply:
Moving to `public/module/k8s/storage-class.ts` is better but few content will be in it